meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool_%.bbappend

```diff
@@ -1,45 +1,6 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-BOOT_TOOLS:mx8-nxp-bsp = "imx-boot-tools"
-
-# From u-boot-imx/meta-freescale
-do_deploy:append:mx8-nxp-bsp() {
-    # Deploy u-boot-nodtb.bin and XX.dtb for mkimage to generate boot binary
-    if [ -n "${UBOOT_CONFIG}" ]; then
-        for config in ${UBOOT_MACHINE}; do
-            machine_idx=$(expr $machine_idx + 1);
-            for type in ${UBOOT_CONFIG}; do
-                type_idx=$(expr $type_idx + 1);
-                if [ $type_idx -eq $machine_idx ]; then
-                    install -d ${DEPLOYDIR}/${BOOT_TOOLS}
-                    # When sign is enabled the final DTB should be copied from deploy dir
-                    if [ "${UBOOT_SIGN_ENABLE}" = "1" ]; then
-                        install -m 0644 ${DEPLOY_DIR_IMAGE}/${UBOOT_DTB_IMAGE} ${DEPLOYDIR}/${BOOT_TOOLS}/${UBOOT_DTB_NAME}
-                    else
-                        install -m 0644 ${B}/${config}/arch/arm/dts/${UBOOT_DTB_NAME} ${DEPLOYDIR}/${BOOT_TOOLS}
-                    fi
-                    install -m 0644 ${B}/${config}/spl/u-boot-spl-nodtb.bin ${DEPLOYDIR}/${BOOT_TOOLS}/u-boot-spl-nodtb.bin-${MACHINE}-${UBOOT_CONFIG}
-                    install -m 0644 ${B}/${config}/u-boot-nodtb.bin ${DEPLOYDIR}/${BOOT_TOOLS}/u-boot-nodtb.bin-${MACHINE}-${UBOOT_CONFIG}
```

Comment:
the only difference between `u-boot-fio-mfgtool_%.bbappend` and `u-boot-fio_%.bbappend` was `s/${UBOOT_CONFIG}/${type}/g`, not sure if I can use `${type}` for both

@ricardosalveti

Reply:
Should be fine to have type for both as it iterates over UBOOT_CONFIG, and in our case we usually have only one config.